### DISCUSSION TITLE
which version of labelme support create ai polygon

### DISCUSSION THREAD

**Discussion by pradeep716** [wkentaro/labelme#1382] [Q&A / Help]
trying to access crate Ai polygon function in m labelme but not showing option there m using  labelme 5.2.1
can anyone help how can i access create ai polygon option or this option have some different version of labelme 

Thanks in advance

**Comment by wkentaro** [comment]
@pradeep716 AI-Polygon feature is included after v5.3.0a0.

![image](https://github.com/labelmeai/labelme/assets/4310419/36ed23e5-38ec-48dd-9077-e31e78ec2dbb)

So v5.3.0a0, v5.3.0, v5.3.1, v5.4.0a0 all include it.

FYI, v5.4.0a0 includes AI-Mask, an another AI-annotation mode.
